platform/vcs-impl/src/com/intellij/openapi/vcs/update/ActionInfoEx.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2000-2017 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intellij.openapi.vcs.update;
+
+import com.intellij.openapi.progress.PerformInBackgroundOption;
+
+/**
+ * Created by kirylch on 4/12/2017.
+ */
+public interface ActionInfoEx extends ActionInfo {
+  static boolean activeToolWindowOnUpdate(ActionInfo action) {
```

Comment:
No, they are non-sticky by default. `VcsNotifier#STANDARD_NOTIFICATION` group is used. 

Note, that there won't be tons of notification even if sticky anyway, because they are merged.

However, receiving even lightweight notifications for a silent action is not a good idea, so it seems that we need some customization here anyway.